src/js/global/fields.js

```diff
@@ -84,7 +84,51 @@ $(document).ready(function(){
         return wprIsParentActive( $parent.closest( '.wpr-field' ) );
     }
 
-    // Display/Hide childern fields on checkbox change.
+	/**
+	 * Masks sensitive information in an input field by replacing all but the last 4 characters with asterisks.
+	 *
+	 * @param {string} id_selector - The ID of the input field to be masked.
+	 * @returns {void} - Modifies the input field value in-place.
+	 *
+	 * @example
+	 * // HTML: <input type="text" id="creditCardInput" value="1234567890123456">
+	 * maskField('creditCardInput');
+	 * // Result: Updates the input field value to '************3456'.
+	 */
+	function maskField(proxy_selector, concrete_selector) {
+		var concrete = {
+			'val': concrete_selector.val(),
+			'length': concrete_selector.val().length
+		}
+
+		if (concrete.length > 4) {
+
+			var hiddenPart = '*'.repeat(Math.max(0, concrete.length - 4));
+			var visiblePart = concrete.val.slice(-4);
+
+			// Combine the hidden and visible parts
+			var maskedValue = hiddenPart + visiblePart;
+
+			proxy_selector.val(maskedValue);
+		}
+
+		proxy_selector.on('input', function () {
+			var proxyValue = proxy_selector.val();
+
+			// Check if the proxy value contains '*' before updating the concrete field
+			if (proxyValue.indexOf('*') === -1) {
```

Comment:
could we use the same character as in password field instead of the `*`? (I don't know the name of it) I think it would look better and maintain consistency

Reply:
Done :)